lib/Notification/styles.scss

```diff
@@ -1,9 +1,7 @@
 .notification {
-	font-size: 13px;
-	line-height: 1.75em;
-	padding: 10px 15px;
+	font-size: $typo-size-slight;
+	padding: $layout-spacing-base/2 $layout-spacing-base*0.75;
```

Comment:
I suggest we use the Style as Config approach I mention in #35. I will amend the styles to match that approach.